Source/WebCore/Configurations/AllowedSPI.toml

```diff
@@ -71,5 +71,5 @@ selectors = [
 request = "rdar://161241270"
 cleanup = "rdar://161241477"
 symbols = [
-    "CTFontDescriptorCreateMatchingFontDescriptorsWithOptions",
-]
\ No newline at end of file
+    "CTFontGetUIFontType",
```

Comment:
I confirmed the request/cleanup radars now reflect the `CTFontGetUIFontType` SPI -> API request, and that this is on track.